sql/core/src/test/scala/org/apache/spark/sql/streaming/StreamSuite.scala

```diff
@@ -211,6 +217,7 @@ class StreamSuite extends StreamTest {
       CheckSinkLatestBatchId(1),
       // Add some data in batch 2
       AddData(inputData, 7, 8, 9),
+      EnsureManualClockInWaitingState,
```

Comment:
Since this test is the only place using `AdvanceManualClock`, I agree that we should merge them together. We can split them in future if necessary. Updating it.